RoslynPluginGenerator/SonarQube.Plugins.Roslyn.PluginGenerator.csproj

```diff
@@ -107,7 +107,6 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="AnalyzerFinder.cs" />
```

Comment:
Did you delete the resources that were used by this class that are no longer required?